### ISSUE TITLE
Write block your own visits extension for Opera

### ISSUE THREAD

**Issue by adriaanvanrossum** [simpleanalytics/roadmap#26]
We have it for Chrome, let's make it work for Opera. https://docs.simpleanalytics.com/extension

**Comment by adriaanvanrossum** [comment]
Inform https://twitter.com/pixelwerk when done. Bram requested this.

**Comment by LeCodeCo** [comment]
Opera should support extensions from the Chrome Web Store, the same way that Microsoft Edge Beta does.

**Comment by adriaanvanrossum** [comment]
Yes, we have the Firefox version coming up anytime soon. I will prioritize the others as well. Seems like a good marketing thing to be in those stores as well + our customers love them.
